### ISSUE TITLE
What's the plan with all the old issues?

### ISSUE THREAD

**Issue by SuperSandro2000** [element-hq/synapse#6]
Since the old repo just got archived, what's the plan element is having with all the issues from the old repo? There are surely still many which still apply and are not fixed, but some are probably rotten and can be kept closed.

**Comment by wrjlewis** [comment]
Good question. We're presented with a set of less than perfect scenarios to sort out issues. Unfortunately there isn't a simple way to transfer issues across orgs and GitHub does not support such an action. 

At the very least we're going to ensure that there isn't a numbering clash between the archived matrix-org/synapse issues and element-hq/synapse, to avoid confusion for folk. We're currently in the process of setting that up. 

After that we will consider the option of pragmatically shifting the content over from previous issues to this repo, using the GitHub API, but since its not an officially supported action by GitHub there will be compromises and it may not be possible to the extent we'd like.

**Comment by TommyTran732** [comment]
> Good question. We're presented with a set of less than perfect scenarios to sort out issues. Unfortunately there isn't a simple way to transfer issues across orgs and GitHub does not support such an action.
> 
> At the very least we're going to ensure that there isn't a numbering clash between the archived matrix-org/synapse issues and element-hq/synapse, to avoid confusion for folk. We're currently in the process of setting that up.
> 
> After that we will consider the option of pragmatically shifting the content over from previous issues to this repo, using the GitHub API, but since its not an officially supported action by GitHub there will be compromises and it may not be possible to the extent we'd like.

How about this:

- Delete this repo
- Transfer the old repo from Matrix.org over
- Have Matrix.org fork the repo and archive it

That way you we get to keep all of the PRs and Issues

**Comment by 6543** [comment]
:point_up: should be done soo ... as it blocks a lot of development :)

**Comment by joshsimmons** [comment]
Hi @TommyTran732, I can speak to that as the Managing Director of the Matrix.org Foundation. We at the Foundation felt it would be improper for us to transfer any of the projects we steward to a for-profit entity, Element or otherwise, in terms of the mission as well as the rules and laws that govern the Foundation.

For sure, it's inconvenient but the Foundation's credibility and tax status both hinge on our independence.

Thankfully, as @wrjlewis noted, the team at Element has come up with some ways to migrate over as much as possible.

Sorry for the added friction!

**Comment by wrjlewis** [comment]
> Thankfully, as @wrjlewis noted, the team at Element has come up with some ways to migrate over as much as possible.

Sorry for the inconvenience and thanks for your understanding 🙏

We've completed 'securing' the numbering between matrix-org/synapse and element-hq/synapse - which, let me tell you, has been a bit of an epic saga due to some mysterious GitHub rate limiting shenanigans. But we got there over the course of many days..

Right now, there's a script in the works running through and transferring content—descriptions, titles, labels, and a link back to the original issue in matrix-org. If all goes to plan, we should have that all wrapped up in a few hours.
